Alameen688/MyDiary
# #159361318 Write test for all basic endpoints

#### What does this PR do?
Delivers the addition of tests to all basic routes
#### Description of Task to be completed?
Ability to test the endpoints that have been implemented 
#### How should this be manually tested?
After cloning the repo, cd into it then
- make sure you have postgresql installed on your machine and start it.
- add a postgreSQL DATABASE_URL to your .env file
   ```
     DATABASE_URL=postgresql://username:password@localhost:5432/mydiarydb
   ```
- The RUN `npm run migrate` to create the database tables.
- Then RUN `npm run test`
#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?
#159361318
